diff --git a/ggml/src/ggml-vulkan/ggml-vulkan.cpp b/ggml/src/ggml-vulkan/ggml-vulkan.cpp
index b62a1c52..9c23a173 100644
--- a/ggml/src/ggml-vulkan/ggml-vulkan.cpp
+++ b/ggml/src/ggml-vulkan/ggml-vulkan.cpp
@@ -4870,7 +4870,7 @@ static void ggml_vk_build_pipelines(ggml_backend_vk_context * ctx) {

     // the number of rows computed per shader depends on GPU model and quant
     uint32_t rm_stdq = 1;
-    uint32_t rm_kq = 2;
+    uint32_t rm_kq = 1;
     uint32_t rm_stdq_int = 1;
     uint32_t rm_kq_int = 1;
     auto const &rm_iq_int = [](uint32_t i) { return i == 0 ? 8u : 4u; };
